wolox-books/src/store.js

```diff
@@ -18,6 +23,19 @@ export default new Vuex.Store({
     },
     userLoggedOut(context) {
       context.commit('setUserStatus', false)
+    },
+    bringBooks(context) {
+      getListOfBooks().then(response => {
+        context.commit('setBooks', response.data)
+      })
+    }
+  },
+  getters: {
+    allBooks(state) {
+      return state.booksList
```

Comment:
Since this are pure functions, use arrow functions instead of methods. One liners are better :smile: